components/movementsensor/replay/replay_test.go

```diff
@@ -764,8 +764,8 @@ func TestUnimplementedFunctionAccuracy(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 
 	acc, err := replay.Accuracy(ctx, map[string]interface{}{})
-	test.That(t, err, test.ShouldResemble, movementsensor.ErrMethodUnimplementedAccuracy)
-	test.That(t, acc, test.ShouldBeNil)
+	test.That(t, err, test.ShouldResemble, nil)
+	test.That(t, acc, test.ShouldNotBeNil)
```

Comment:
[optional] Is it worth testing that we got NaN and the other invalid values here? I can go either way.